_widget/src/components/app/component.vue

```diff
@@ -52,17 +49,20 @@ export default {
       default(url) {
         return window.fetch(url).then((r) => r.json());
       }
+    },
+    search: {
+      type: Object,
+      default() { return new Search(); }
     }
   },
   data () {
-    const query = this.query || urlMatchingDictionary(window.location.href);
```

Comment:
We weren't using `query` and I thought it was a good name for the searching method. 🤷‍♂️